cmd/bosun/conf/conf.go

```diff
@@ -31,7 +31,9 @@ import (
 // outside of the package without a setter
 type SystemConfProvider interface {
 	GetHTTPListen() string
-	GetRelayListen() string
+	GetHTTPSListen() string
+	GetTLSCertFile() string
```

Comment:
This is to correspond to [http.ListenAndServeTLS](https://golang.org/pkg/net/http/#ListenAndServeTLS) which expects two key file locations. They are pem encoded, yes, but maybe you mean a combined keyfile of some kind. There are other ways to host a tls server, but this is the easiest.